src/App.vue

```diff
@@ -153,6 +153,13 @@ export default {
     let config;
     try {
       config = await this.getConfig();
+      const path = (window.location.pathname != '/') ? window.location.pathname : null;
+      if (path) {
+        let pathConfig = await this.getConfig(`assets${path}.yml`); // the slash (/) is included in the pathname
```

Comment:
When using a query string the slash is no longer part of the pathname and needs to be added.
```suggestion
        let pathConfig = await this.getConfig(`assets/${path}.yml`);
```